src/js/views/maps/cartodb-layer.js

```diff
@@ -88,7 +88,8 @@ define(function (require, exports, module) {
         var url = 'https://' + data.cdn_url.https +
             '/localdata/api/v1/map/' + data.layergroupid +
             '/{z}/{x}/{y}.png';
-        this.tileLayer = L.tileLayer(url);
+        var attribution = this.layerOptions.attribution || 'hello world';
```

Comment:
If there's no attribution specified, can we leave it blank, instead of using `'hello world'`? We can probably pass `undefined` along and get the appropriate behavior.